test/Dialect/Torch/canonicalize.mlir

```diff
@@ -2146,3 +2146,9 @@ func.func @torch.aten.masked_fill.Tensor$canonicalize(%arg0: !torch.vtensor<[?,?
   %1 = torch.aten.masked_fill.Tensor %arg0, %arg1, %0 : !torch.vtensor<[?,?],f32>, !torch.vtensor<[?,?],i1>, !torch.vtensor<[],f32> -> !torch.vtensor<[?,?],f32>
   return %1 : !torch.vtensor<[?,?],f32>
 }
+
+// CHECK-LABEL:   func.func @torch.aten.detach$canonicalize
```

Comment:
Did you intend to add a `CHECK-NEXT` for `torch.aten.detach` (to confirm it didn't get folded by `-canonicalize`)?